skypy/halos/tests/test_sampler.py

```diff
@@ -0,0 +1,32 @@
+import numpy as np
+import colossus as colossus
+from colossus.lss import mass_function
+from scipy import integrate
+from scipy.stats import kstest
+import pytest
+
```

Comment:
```suggestion
from skypy.halos.sampler import HAS_COLOSSUS

```
We want to import the `HAS_COLOSSUS` variable so we know if colossus is installed or not